compiler/build.gradle

```diff
@@ -150,6 +154,11 @@ protobuf {
     ofSourceSet('test')*.plugins {
       grpc {}
     }
+    ofSourceSet('testLite')*.plugins {
+      grpc {
+        option 'lite'
```

Comment:
I had it originally, and then removed it from this PR, since that means changing some other documentation as well. I think I'd still like to do it separately. 'lite' is the flag we are expecting to use for protoc itself (and that I see internally).